packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -58,8 +58,8 @@ import {
 	createDocumentId,
 	waitForContainerConnection,
 } from "@fluidframework/test-utils/internal";
-import { SchemaFactory, TreeConfiguration } from "@fluidframework/tree";
-import { ISharedTree, SharedTree } from "@fluidframework/tree/internal";
+import { SchemaFactory } from "@fluidframework/tree";
+import { ISharedTree, SharedTree, TreeViewConfiguration } from "@fluidframework/tree/internal";
```

Comment:
It got imported from /internal in a few places but I see it's a public API already and got imported from "@fluidframework/tree" in others, so should be from the public entrypoint everywhere, right?